### PR TITLE
9.0 add partner changeset

### PR DESCRIPTION
**Pull request by jt-xx**

### CODE REVIEW

**Comment by guewen** [comment]
Hi @jt-xx, you should cherry-pick this correction: https://github.com/OCA/partner-contact/commit/cdc73214a3c6d6f7a74b77ff241a725168fd26dc

File: partner_changeset/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{'name': 'Partner Changesets',
+ 'version': '8.0.1.0.0',
```
Comment (yvaucher) [inline partner_changeset/__openerp__.py:6]: Version to change to 9.0.1.0.0

**Comment by yvaucher** [comment]
@jt-xx Seeing your last commit I set this as WIP

**Comment by pedrobaeza** [comment]
Superseeded by #357

**Pull request closed without merging**